src/spikeinterface/sortingcomponents/merging/__init__.py

```diff
@@ -0,0 +1 @@
+from .main import merge_spikes
```

Comment:
I would remove this and use only the curation module.
I think we all the machinery there.